sdks/python/apache_beam/runners/interactive/interactive_runner.py

```diff
@@ -259,7 +247,8 @@ def read(self, pcoll, include_window_info=False):
     WindowedValues. Otherwise, return the element as itself.
     """
     key = self._pipeline_instrument.cache_key(pcoll)
-    cache_manager = ie.current_env().cache_manager()
+    cache_manager = ie.current_env().get_cache_manager(
+        self._pipeline_instrument.user_pipeline)
     if cache_manager.exists('full', key):
```

Comment:
Do you need to check that `cache_manager` is not `None`?

Reply:
That is not needed since PipelineResult takes in the PipelineInstrument in the constructor. The cache manager must have been initialized during the instrumenting.